.storybook/stories/combobox/combobox.stories.ts

```diff
@@ -71,28 +71,49 @@ const defaultParameters: Parameters = {
     // story helpers
     elements: { control: { disable: true }, table: { disable: true } },
     optionCount: { control: { type: 'number', min: 1, max: elements.length } },
-    formControl: { control: { disable: true }, table: { disable: true } },
   },
   args: {
     // outputs
     clrInputChange: action('clrInputChange'),
     clrOpenChange: action('clrOpenChange'),
     clrSelectionChange: action('clrSelectionChange'),
     // story helpers
-    formControl: new FormControl(),
     optionCount: elements.length,
     content: 'Option',
     controlDisabled: false,
     elements,
+    singleModel: 'Am',
+    multiModel: ['Am', 'As', 'Ba'],
+    isDisabled: null,
```

Comment:
We still have `controlDisabled`, and this arg isn't typed as a boolean.

![Screenshot_20230806_231208](https://github.com/vmware-clarity/ng-clarity/assets/7399499/4f9be4d2-7fef-4c80-93c3-5eb3e4b07fb9)

Reply:
Good catch, moved back to controlDisabled